dropwizard-servers/src/main/java/com/palantir/remoting1/servers/dropwizard/DropwizardServers.java

```diff
@@ -44,5 +41,6 @@ public void accept(ExceptionMapper<? extends Throwable> mapper) {
                     }
                 });
         environment.jersey().register(new OptionalAsNoContentMessageBodyWriter());
+        environment.jersey().register(new TraceEnrichingFilter());
```

Comment:
also this whole class can go away now that there's nothing Dropwizard specific left?